### PR TITLE
Upgrade to generator-jhipster@6.8.0

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Fix https://github.com/jhipster/jhipster-vuejs/issues/574

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/jhipster-vuejs/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
I can reproduce the failure locally:
- in generator-jhipster: npm link
- in jhipster-vuejs: npm link
- in jhipster-vuejs: npm test -> it failed

Maybe you can help @mshima ?

**Comment by mshima** [comment]
@pascalgrimaud there are 2 possible fixes:
- Remove dependency on yeoman-generator and yeoman-environment, I think they aren’t used.
- Move them to dev dep and update if tests fails. Yeoman-test maybe brings an older version of them.

**Comment by pascalgrimaud** [comment]
thanks a lot, I'll probably try to fix it next week.

**Comment by mshima** [comment]
I will try to take a look tomorrow.

**Comment by mshima** [comment]
@pascalgrimaud tty error is a Inquirer bug related to https://github.com/SBoudrias/Inquirer.js/pull/891

**Comment by pascalgrimaud** [comment]
@mshima : should I force the use of specific version of inquirer ?
Then, in main generator, we have https://github.com/jhipster/generator-jhipster/blame/master/package.json#L103 -> 9 months ago

**Comment by mshima** [comment]
> @mshima : should I force the use of specific version of inquirer ?
> Then, in main generator, we have https://github.com/jhipster/generator-jhipster/blame/master/package.json#L103 -> 9 months ago

I think we should update main to 7.0.4 too.
I just created a new PR on inquirer for a new feature.

**Comment by mshima** [comment]
The problem only happens on inquirer + yeoman-test :
https://github.com/yeoman/yeoman-test/issues/78

**Comment by mshima** [comment]
@pascalgrimaud 7.0.4, I think 7.0.6 didn’t fixed it.
Let’s wait the CI status.

**Comment by pascalgrimaud** [comment]
thanks again @mshima !

File: package-lock.json [hunk 1]
```diff
@@ -7655,956 +6817,406 @@
...
+          "version": "4.0.0",
+          "resolved": "https://registry.npmjs.org/has-flag/-/has-flag-4.0.0.tgz",
+          "integrity": "sha512-EykJT/Q1KjTWctppgIAgfSO0tKVuZUjhgMr17kqTumMl6Afv3EISleU7qZUzoXDFTAHTDC4NOoG/ZxU3EvlMPQ==",
+          "dev": true
         },
         "inquirer": {
-          "version": "6.5.2",
-          "resolved": "https://registry.npmjs.org/inquirer/-/inquirer-6.5.2.tgz",
-          "integrity": "sha512-cntlB5ghuB0iuO65Ovoi8ogLHiWGs/5yNrtUcKjFhSSiVeAIVpD7koaSU9RM8mpXw5YDi9RdYXGQMaOURB7ycQ==",
+          "version": "7.0.6",
```
Comment (mshima) [inline package-lock.json]: It’s loading 7.0.6 here

Comment (pascalgrimaud) [reply package-lock.json]: What can I do to force the 7.0.4 ? lol

Comment (mshima) [reply package-lock.json]: 😂 
Good question. I am on mobile now.
I didn’t even knew about resolutions.

Comment (mshima) [reply package-lock.json]: Looks like https://www.npmjs.com/package/npm-force-resolutions is needed to make resolutions work.

Comment (pascalgrimaud) [reply package-lock.json]: Anyway, Inquirer is related to TTY problem, isn't it ? So why the main generator doesn't have the problem ? Only the blueprint ?

Comment (mshima) [reply package-lock.json]: The main generator has inquirer locked by package-lock to others versions.

**Comment by pascalgrimaud** [comment]
@mshima : I merged your PR, but Vue.js blueprint uses exact 6.8.0 and not master

If https://github.com/jhipster/generator-jhipster/pull/11453 is absolutely necessary, I can't release Vue.js blueprint without a patch release of generator-jhipster 6.8.1

**Comment by pascalgrimaud** [comment]
thanks @mshima for your help
there are 2 last issues:
- embedded document, for couchbase and kafka
- increase coverage

**Comment by pascalgrimaud** [comment]
I'm merging this, as the 2 last items can be done in separate PRs

**Comment by mshima** [comment]
> @mshima : I merged your PR, but Vue.js blueprint uses exact 6.8.0 and not master
> 
> If [jhipster/generator-jhipster#11453](https://github.com/jhipster/generator-jhipster/pull/11453) is absolutely necessary, I can't release Vue.js blueprint without a patch release of generator-jhipster 6.8.1

I don’t think it’s needed. This problem is old and I forgot to create a PR to fix it.
https://github.com/mshima/generator-jhipster/commit/4bd86209e77ff61b7431c69fc5fa6aa0536624b2
CI succeeded now.

**Comment by mshima** [comment]
By the way inquirer 7.0.7 is released to fix yeoman-test.

**Pull request merged**